flatliners/comparisonscore.py

```diff
@@ -51,4 +53,11 @@ def compute_cluster_distance(self, values):
         # and then take the square root of the sum to calculate the Euclidean distance between vectors.
         # store final, single value for each cluster in scores.
         self.clusters[cluster_id][resource] = (value - self.versions[version_id][resource])**2
-        self.score[cluster_id] = {'cluster': cluster_id, 'score': (sum(list(self.clusters[cluster_id].values())))**0.5 }
\ No newline at end of file
+        self.score[cluster_id] = {'cluster': cluster_id, 'score': (sum(list(self.clusters[cluster_id].values())))**0.5 }
+    
+     def wait_for_resources(self, x):
+        cluster_id = x['cluster']
+        if len(self.clusters[cluster_id].keys()) == 63:
```

Comment:
what happens, if for some reason the number of keys jump from 62 to 64? You'll never return. For checks like this, it's generally safer to check for `>= 63`
Also, at least a comment why it's 63?

Reply:
@durandom, working knowledge of this particular data set has shown that there are 64 resources tracked by `etcd_objects resources`. So this number shouldn't jump without someone one knowing about it. But I agree, a hard coded value is probably not the best approach here.

The main reason for this method is to simply prevent the "weirdness score" from behaving erratically, while the first round of data points are populating the initial array. perhaps a better solution would be to check if a value is being added, in which case we will not publish, or if a value is being updated, in which case we will publish. This should achieve the same goal, independent of the number of features.

Reply:
I'm sure this number will jump from version to version. It's arbitrary of what is being tracked in etcd.